### PR TITLE
Fix instructor bio editing capability for all internal users.

### PR DESCRIPTION
**Pull request by attiyaIshaque**
### Description
This PR fixes that all internal users would be able to edit the instructor.

### Testing
**https://discovery-escalation-discovery.sandbox.edx.org/publisher/course_runs/7/edit/**
username: Publisher_Attiya
password: Attiya

You would be able to edit the instructor info.

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [x] @asadazam93 
- [x] @tasawernawaz 

### Post-review
- [x] Rebase and squash commits

### CODE REVIEW

**Review by asadazam93 — APPROVED** [review]

File: course_discovery/apps/publisher/forms.py [hunk 1]
```diff
@@ -29,6 +29,7 @@ def label_from_instance(self, obj):
         context = {
             'profile_image': obj.get_profile_image_url,
             'full_name': obj.full_name,
+            'can_edit_instructor': obj.uuid if not obj.profile_image_url else None,
             'uuid': obj.uuid,
```
Comment (tasawernawaz) [inline course_discovery/apps/publisher/forms.py]: you have already `uuid` here, why we need to save it again in `can_edit_instructor`.

File: course_discovery/apps/publisher/templates/publisher/_personFieldLabel.html [hunk 2]
```diff
@@ -1 +1 @@
-<img src="{{ profile_image }}"/><span{% if uuid %} data-uuid="{{ uuid }}" {% endif %}>{{ full_name }}</span>{% if organization_id %}<span>{{ organization_id }}</span>{% endif %}
+<img src="{{ profile_image }}"/><span{% if can_edit_instructor %}  data-can-edit="{{ can_edit_instructor }}" {% endif %} data-uuid="{{ uuid }}" >{{ full_name }}</span>{% if organization_id %}<span>{{ organization_id }}</span>{% endif %}
```
Comment (tasawernawaz) [inline course_discovery/apps/publisher/templates/publisher/_personFieldLabel.html:1]: possibly both 'data-can-edit' and 'data-uuid' will have same value. Can you use `True/Flase` for possible values of `can_edit_instructor` ?

File: course_discovery/apps/publisher/templates/publisher/_personLookup.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<table class="instructor-option" id="{{ uuid }}">
+<table class="instructor-option" id="{{ uuid }}" {% if can_edit_instructor %}  data-can-edit="{{ can_edit_instructor }}" {% endif %}>
```
Comment (tasawernawaz) [inline course_discovery/apps/publisher/templates/publisher/_personLookup.html:1]: same here

**Review by tasawernawaz — CHANGES_REQUESTED** [review]

**Comment by tasawernawaz** [comment]
@attiyaIshaque If sandbox is ready please update the description with sandbox link.

File: course_discovery/apps/publisher/templates/publisher/_personFieldLabel.html (hunk 2 above)
Comment (attiyaIshaque) [reply course_discovery/apps/publisher/templates/publisher/_personFieldLabel.html:1]: I got your point. Thanks, I will change it. :)

**Comment by attiyaIshaque** [comment]
@tasawernawaz I have added the sandbox link.

File: course_discovery/apps/course_metadata/lookups.py [hunk 4]
```diff
@@ -72,7 +72,8 @@ def get_result_label(self, result):
                 'profile_image': result.get_profile_image_url,
                 'full_name': result.full_name,
                 'position': result.position if hasattr(result, 'position') else None,
-                'organization_id': result.position.organization_id if hasattr(result, 'position') else None
+                'organization_id': result.position.organization_id if hasattr(result, 'position') else None,
+                'can_edit_instructor': True if not result.profile_image_url else False,
```
Comment (tasawernawaz) [inline course_discovery/apps/course_metadata/lookups.py]: you can replace `True if not result.profile_image_url else False` with `not result.profile_image_url`

**Review by tasawernawaz — APPROVED** [review]
Just a minor change requested.

**Comment by attiyaIshaque** [comment]
@tasawernawaz I have addressed your requested change. Now I am merging it. Thanks

**Pull request merged**